tests/unit_tests/test_keyfile.py

```diff
@@ -615,10 +615,12 @@ def test_get_coldkey_password_from_environment(monkeypatch):
         "WALLET": "password",
         "my_wallet": "password2",
         "my-wallet": "password2",
+        "my-wallet3": "password3",
     }
 
     monkeypatch.setenv("bt_cold_pw_wallet", password_by_wallet["WALLET"])
     monkeypatch.setenv("BT_COLD_PW_My_Wallet", password_by_wallet["my_wallet"])
+    monkeypatch.setenv("BT_COLD_PW_My-Wallet3", password_by_wallet["my-wallet3"])
```

Comment:
won't the new warning crash this?

Reply:
CI is passing. Test are not run with `pytest -W error` if this is what you are referring to.

Or did I misunderstand you and you mean something else?